### ISSUE TITLE
Not able to load RTF files

### ISSUE THREAD

**Issue by saeynic** [Sicos1977/IFilterTextReader#16]
Dear Kees,

I have an issue with RTF files, it consequently fails on each RTF file with a COMException on this line:
`iPersistStream.Load(comStream);`
In attach I have included more background information. Could you please have a look at this? 

[Debug_Info.zip](https://github.com/Sicos1977/IFilterTextReader/files/275517/Debug_Info.zip)

Many thanks!

Best regards.
Nico

FYI: I'm using Windows 10 Build 10240 64 bit and your latest version 1.5.0.0

**Comment by Sicos1977** [comment]
Hi,

Your RTF seems to work fine overhere... Try reinstalling your Office IFilters --> https://www.microsoft.com/en-us/download/details.aspx?id=17062

![21-5-2016 08-01-17](https://cloud.githubusercontent.com/assets/6692947/15446743/6448cf5c-1f2a-11e6-9bb3-305121302c74.png)

And since your dutch :-) ... met vriendelijke groet,
Kees

**Comment by saeynic** [comment]
Kees,

Thank you for this tip. I reinstalled the Office filters, but only the Microsoft Office Filter got a higher version. the other filter DLL files (and the rtffilt.dll) had the same version as before: (FYI:)
![searchfilterview2](https://cloud.githubusercontent.com/assets/12798297/15451059/4d9e621c-1fb1-11e6-9e93-5f7b0bc183fc.JPG)

But I did some extra steps, based on this thread:
http://answers.microsoft.com/en-us/windows/forum/windows_7-files/windows-7-will-not-search-my-rtf-files-i-went-into/e7f885dc-1783-e011-9b4b-68b599b31bf5
1. rebuild index (not relevant I guess, but I did it anyway)
2. register with administrator permissions "regsvr32 rtffilt.dll"
3. reboot PC

And... yes -> it worked!

Many thanks!
Best regards from Belgium ;o)
Nico
